extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -28,75 +34,136 @@ public static function onBeforePageDisplay() {
 		return true;
 	}
 
-	public static function render( $input, array $args ) {
-		global $wgCityId;
+	/**
+	 * Checks arguments for errors.
+	 * @param array $args
+	 * @param string errorMessage Return parameter with the proper error message to show. Disregard if return is false
+	 * @return true if ok, false if error
+	 */
+	private function checkArguments( array $args, $modelData, &$errorMessage ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)